Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

```diff
@@ -98,6 +98,24 @@ example.com. IN CAA 0 issuewild "digicert.com"
 
 On the Professional and Enterprise plan, you can manually add your own certificate to your Umbraco Cloud project and bind it to one of the hostnames you've added.
 
+## [Using WAF on Cloud](WAF)
+There is an option to install Web Application Firewall if that is needed.
+If you are using a custom certificate for that specific hostname with WAF make sure you are able to upload the custom certificate:
+
+1. Make sure the hostname is pointing to  Umbraco Cloud (dns. umbraco.io)
+2. Certificates are issued for the actual hostname - It is probably required to have a custom certificate for a WAF hostname. 
+3. Be on a plan that supports custom certificates
+
+Even though the hostname isn't pointing to Umbraco Cloud (dns.umbraco.io) can the certificate be validated by adding a txt record for the hostname. Or a DNS record.
+When that is validated, then the hostname will, in principle, be ready to be used in Umbraco Cloud.
+
+So when you update your DNS, regarding go-live, will it typically be very fast and the hostname will be ready to use.
```

Comment:
Don't think this is necessary. It becomes a bit confusing as its mixing two different things.